prow-jobs/pingcap-tiproxy-latest-presubmits.yaml

```
@@ -1,15 +1,37 @@
 # struct ref: https://pkg.go.dev/k8s.io/test-infra/prow/config#Presubmit
 presubmits:
   pingcap/tiproxy:
-    - name: pingcap/tiproxy/pull_mysql_test
+    - name: pingcap/tiproxy/ghpr_build
       agent: jenkins
       decorate: false # need add this.
       skip_if_only_changed: "(\\.md|Dockerfile|OWNERS|OWNERS_ALIASES)$"
       run_before_merge: true
-      context: pull-mysql-test
+      context: idc-jenkins-ci-tiproxy/build
```

please keep it simple.
```suggestion
      context: pull-build
```

---

and update other jobs.